src/main/java/no/nav/fo/veilarbdialog/service/DialogDataService.java

```diff
@@ -1,85 +1,78 @@
 import no.nav.fo.veilarbdialog.db.dao.DialogDAO;
 import no.nav.fo.veilarbdialog.db.dao.DialogFeedDAO;
 import no.nav.fo.veilarbdialog.domain.*;
-import no.nav.fo.veilarbdialog.domain.KafkaDialogMelding;
-import no.nav.sbl.featuretoggle.unleash.UnleashService;
-import org.springframework.stereotype.Component;
+import no.nav.fo.veilarbdialog.feed.KvpService;
+import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.Collections;
 import java.util.Date;
 import java.util.List;
 import java.util.Optional;
 
-@Component
+@Service
 @Transactional
-public class AppService {
+@RequiredArgsConstructor
+public class DialogDataService {
 
-    private final AktorService aktorService;
+    private final AktorregisterClient aktorregisterClient;
     private final DialogDAO dialogDAO;
     private final DialogStatusService dialogStatusService;
     private final DataVarehusDAO dataVarehusDAO;
     private final DialogFeedDAO dialogFeedDAO;
-    private final PepClient pepClient;
-    private final KvpClient kvpClient;
+    private final KvpService kvpService;
     private final UnleashService unleashService;
     private final KafkaDialogService kafkaDialogService;
-
-    public AppService(AktorService aktorService,
-                      DialogDAO dialogDAO,
-                      DialogStatusService dialogStatusService,
-                      DataVarehusDAO dataVarehusDAO,
-                      DialogFeedDAO dialogFeedDAO,
-                      PepClient pepClient,
-                      KafkaDialogService kafkaDialogService,
-                      KvpClient kvpClient,
-                      UnleashService unleashService) {
-        this.aktorService = aktorService;
-        this.dialogDAO = dialogDAO;
-        this.dialogStatusService = dialogStatusService;
-        this.dataVarehusDAO = dataVarehusDAO;
-        this.dialogFeedDAO = dialogFeedDAO;
-        this.pepClient = pepClient;
-        this.kvpClient = kvpClient;
-        this.unleashService = unleashService;
-        this.kafkaDialogService = kafkaDialogService;
+    private final AuthService auth;
+
+    private String assertAccessToAktorId(String aktorId)
+            throws IngenTilgang {
+        if (!auth.harTilgangTilPerson(aktorId)) {
+            throw new IngenTilgang(String.format(
+                    "%s har ikke lesetilgang til %s",
+                    auth.getIdent().orElse("null"),
+                    aktorId
+            ));
+        }
+        return aktorId;
     }
 
     @Transactional(readOnly = true)
-    public List<DialogData> hentDialogerForBruker(Person person) {
-        String aktorId = hentAktoerIdForPerson(person);
-        pepClient.sjekkLesetilgangTilAktorId(aktorId);
-
+    public List<DialogData> hentDialogerForBruker(Person person)
+            throws IngenTilgang {
+        String aktorId = assertAccessToAktorId(hentAktoerIdForPerson(person));
```

Comment:
Er det noe rart med indenteringen her?

Reply:
Hva da? Default IntelliJ auto-format her også.

Reply:
Ops, så ikke at det hører til linja over jeg så ble litt forvirret. Er linja for lang hvis du ikke har newline før throws eller er det at du foretrekker å ha det sånn?

Reply:
Begge deler, egentlig.

Forrige prosjekt hadde hardcore formatting regler der man nesten måtte ha (minst) ett linjeskift per metodesignatur...

Tidligere har jeg vært vant til å ha `throws` på egen linje, for å gjøre lesbarheten større.